Quran/HighlightingView.swift

```diff
@@ -12,7 +12,36 @@ import UIKit
 // It's also expected to reuse layers instead of dropping & creating new ones.
```

Comment:
For this class, I would expect a re-draw if any of the properties change. Something like that
```
var highlightedAyat: [AyahNumber] = [] {
  didSet {
    setNeedsDisplay()
  }
}
```
The same for `ayahInfoData`,  `imageScale` and `yOffset`.